sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/TestJsonData.scala

```diff
@@ -231,4 +231,10 @@ private[json] trait TestJsonData {
   lazy val singleRow: RDD[String] = spark.sparkContext.parallelize("""{"a":123}""" :: Nil)
 
   def empty: RDD[String] = spark.sparkContext.parallelize(Seq[String]())
+  
+  def dataset(rdd: RDD[String]): Dataset[String] = {
```

Comment:
actually we don't need this , see https://github.com/apache/spark/pull/16895/files#r100681578